cmd/rook/ceph/mon.go

```diff
@@ -115,7 +115,7 @@ func compareMonSecret(secret, configDir string) error {
 		return nil
 	}
 	if cachedKeyring.Value() != secret {
-		return fmt.Errorf("The keyring does not match the existing keyring in %s. You may need to delete the contents of dataDirHostPath on the host from a previous deployment.", cachedKeyringFile)
+		return fmt.Errorf("the keyring does not match the existing keyring in %s. You may need to delete the contents of dataDirHostPath on the host from a previous deployment", cachedKeyringFile)
```

Comment:
Would you mind reverting the changes to all the error message formatting for sentence structure? Some of them might be a different pattern from other projects, but I'd like to stay consistent within the project. You can remove the newlines though, I'm not sure why they were used. `TerminateFatal()` makes sure to terminate the output with a newline anyway.

Reply:
@travisn I am bit confused with above. Are you saying whatever the changes brought into err strings has to be reverted. The only change in this PR is on 2 types>

[x] - Error strings are not started with a capital letter in it 
[x] - The punctuation has been removed from end of the string , for ex: `.`. 

These 2 changes looks fine to me . But, if you say we have to remove then I can recondier :)

Reply:
Individually these error messages may be ok without punctuation, but in Rook they are frequently aggregated with other errors to form multiple sentences in the same error messages. While it may not match the pattern of other projects or suggestions by linter, it has worked well like this for Rook. Of the changes in this PR, I'm suggesting:
- Revert capitalization and punctuation changes
- Removing the `\n` from the end of error messages is ok. Looks like this is mostly where `TerminateFatal()` is called and we shouldn't need the newline in the err.
- Keep the comments you've added
- Keep the whitespace before the package names

Reply:
Hopefully done .. @travisn ptal :). Thanks